components/StyledTag.js

```diff
@@ -116,7 +116,6 @@ const CloseButton = styled.button.attrs({
   border: none;
   padding: 0;
   line-height: inherit;
-  outline: none;
```

Comment:
Why remove this? If this is just for `StyledInputTags`, can we find a way to not apply the change globally?

Reply:
For accessibility reasons, I think the close button should have the outline. But I do understand if atm it might regress behaviour elsewhere. Any suggestions? As the only way I can see to modify that style is inside `StyledTag` to my knowledge.